### DISCUSSION TITLE
Option to edit stack limit of enderpearls, eggs and snowballs!!

### DISCUSSION THREAD

**Discussion by shateq** [PurpurMC/Purpur#526] [__Dismissed__] [answered]
I suggest a new **config entry** that can lower or adjust the space that eggs can take in the hotbar/inventory. Of course we'll have to make this compatible when 1.18 when mojang will release pouches later (this is the only minus point).

`stack-limit:
  egg: 16
  ender-pearl: 2
  snowball: 32 (default is 16 for each)
`

**Comment by ATMmachin3** [comment]
Pouches were released in 1.17

**Reply by BillyGalbreath** [reply]
Half released. They exist in the game, but have no recipe and are coded to not show up in creative menu until 1.18

You can use the `/give` command or add your own recipe in a datapack/plugin to obtain them in 1.17, though. I added a recipe via datapack on my own server so we can use them now ^_^

**Comment by BillyGalbreath** [comment, marked as answer]
Purpur will never control the max size of itemstacks. There are simply too many quirks which come with changes them (dupes, lost items, other bugs, etc). There are plugins out there that attempt to control the max stack sizes, but they all have issues as well. Use one of those plugins if you really want such a feature.
